python/activator/activator.py

```diff
@@ -65,14 +73,13 @@
 
 app = Flask(__name__)
 
-subscriber = pubsub_v1.SubscriberClient()
-topic_path = subscriber.topic_path(
-    PROJECT_ID,
-    f"{instrument_name}-image",
-)
-subscription = None
+consumer = kafka.Consumer({
+    "bootstrap.servers": kafka_cluster,
+    "group.id": kafka_group_id,
+    "auto.offset.reset": "earliest",
```

Comment:
`earliest` may be overkill.  The desired message for this worker can't show up until after the next_visit event that triggered the worker.  But if cold start takes a while, I guess it's possible that the image arrived during that time, especially for images with shorter exposure times.

Note that the dynamic subscription I was using before for Pub/Sub is the equivalent of `latest`, not `earliest`.

We'll need to expire messages from the Kafka queues relatively rapidly if we don't want to have to search through a lot of old images.  I believe the default is 7 days.

Reply:
This was cargo-culted from [test-consumer](https://github.com/slaclab/rubin-usdf-prompt-processing/blob/main/test-consumer/main.py). (Sorry!)

I think this might be tied to your question below about `subscribe` -- can we actually make assumptions about the relative ordering of messages (`next_visit` vs. bucket notification) transmitted over different channels?

Reply:
I think the worst case scenario is biases, where each exposure might be ~2 seconds in length (as that is the readout time).  next_visit would then be a minimum of 4 seconds before readout finishes (sent prior to the previous bias) and at least 4.5 seconds before the file would be compressed and copied from the CCS (at the Summit; likely more at USDF).

The normal science case with single snaps is at least 64.5 seconds before the first file could arrive; 49.5 seconds for two snaps.

Having the next_visit message fall so far behind that the image notification arrives first thus seems highly unlikely.  But I'm OK with `earliest` if we set the Kafka queues to no more than a day or even an hour of retention.